### DISCUSSION TITLE
How do I enable the menu background to use solid themes?

### DISCUSSION THREAD

**Discussion by TheTraygon** [Godiesc/firefox-one#31] [Q&A] [answered]
I'm not a dev and don't plan on being one, so I apologize if this is barebones basic stuff, but I've taken the image I want and named it as `menu-image.png`, but nothing. I'm afraid I had to go into the code and mess with some stuff. Any help is appreciated!

**Comment by Godiesc** [comment]
you have to add this preference in about:config page and set it true, if you don't know what is about:config you should look this link: [Firefox-help](https://support.mozilla.org/en-US/kb/about-config-editor-firefox)

![imagen](https://github.com/Godiesc/firefox-one/assets/22057609/77c02da4-3e17-4cf0-a7c7-9f8e9fd8a58b)

**Reply by TheTraygon** [reply]
I have done that already and it didn't work

**Reply by Godiesc** [reply]
tested just now and the image display correctly, be sure your new image is in the right format and into the right directory.

![imagen](https://github.com/Godiesc/firefox-one/assets/22057609/f152d7b7-4ffe-42d6-9015-099abff2352d)

**Comment by TheTraygon** [comment, marked as answer]
Alright I don't know what changed but I did a fresh install with all extra features pre-configed and that seemed to work
